projects/managed-hubs/support.md

```diff
@@ -231,3 +231,49 @@ Support Budget
 
   :::
 ```
+
+(support:expected-time-off)=
+## Expected time off and downtime
+
+There are some cases when we intentionally reduce our operations and support capacity.
+See [](time-off:annual-expected) for our broader policies and support commitments during this time.
+
+(support:expected-time-off:policy)=
+### Expected time off policy
+
+Below is our policy for support during expected time off:
+
+- We monitor our support e-mail for major incidents, but do not guarantee a response for non-essential requests or questions.
+- We will take steps to minimize harm and avoid catastrophic problems (e.g. incidents that drastically increase costs), but will not perform non-essential actions. We will not treat "a general outage" as catastrophic unless we get an explicit signal in advance from a community representative they are doing essential work during this time.
```

Comment:
How much time does it mean "in advance" here?

Reply:
I am not sure, I struggled to define this without getting into too much detail. What I'm trying to convey is the idea that in some cases, we might get a community that makes an explicit request because some really important thing is happening. I wanted to suggest that we are open to conversation about this if it is really a problem for them.

Maybe it would be simpler to just say "sorry but we still can't promise anything" for now?

Reply:
> makes an explicit request because some really important thing is happening

This makes sense. Maybe one approach could be ask for this request to come before everyone leaves on vacation and there's enough time for everyone to plan around this (maybe 1, 2 weeks before)?

> Maybe it would be simpler to just say "sorry but we still can't promise anything" for now?

This approach makes me nervous because I feel like this policy translates into:
    *"we shouldn't expect anything from the support team, but we can hope they do everything"*

Which (for me at least) produces more stress than knowing exactly what are my responsabilities. Not sure if this makes any sense.

Reply:
> (maybe 1, 2 weeks before)?

2 weeks at minimum, I would say...

> Which (for me at least) produces more stress than knowing exactly what are my responsabilities. Not sure if this makes any sense.

I think the responsibilities during this time are generally well outlined in the above list. I do think we need to change our mindset to accommodate the vision on that list (I know it is not easy!!).